rhessys/util/readtag_worldfile.c

```diff
@@ -89,8 +89,9 @@ param *readtag_worldfile(int *paramCnt, FILE *file,char *key){
 	    if(strcmp(strbuf2,"n_basestations")==0){
 	      break;
 	    }
+	    
```

Comment:
Brian,
     I only test it on a 30MB worldfile and the performance is identical. I
haven't got any large worldfile to test it.

X

On Fri, Mar 25, 2016 at 1:06 PM, Brian Miles <notifications@github.com>
wrote:

> In rhessys/util/readtag_worldfile.c
> <https://github.com/RHESSys/RHESSys/pull/82#discussion_r57484822>:
>
> > @@ -89,8 +89,9 @@ param *readtag_worldfile(int *paramCnt, FILE *file,char *key){
> >  	    if(strcmp(strbuf2,"n_basestations")==0){
> >  	      break;
> >  	    }
> > +	
>
> On the other hand, if you have tested this on a reasonably large worldfile
> (a few GB), and the performance for a 1 day model run is within ~5% of the
> previous version it might not be worth worrying about. Up to you.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/RHESSys/RHESSys/pull/82/files/e0ec292f00e9aaf33834fad68ab9df8d5e9ada61#r57484822>
>



-- 
Xiaoli Chen

Bren School of Environmental Science and Management
University of California, Santa Barbara
xiaoli_chen@umail.ucsb.edu